src/main/resources/db/changelog/0003-create-table-financial_transaction_categories.yaml

```diff
@@ -26,5 +26,17 @@ databaseChangeLog:
               - column:
                   name: transaction_type
                   type: varchar(255)
+                  constraints:
+                    nullable: false
+  - changeSet:
+      id: "add column user_id to financial transaction categories"
+      author: SzanieckiFranciszek
+      changes:
+        - addColumn:
+            tableName: financial_transaction_categories
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
                   constraints:
```

Comment:
Why didn't you implement this as a new databaseChangeLog?

Reply:
Corrected. I Created new file: 0012-add-new-column-to-financial-transaction-category.yaml.